misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/JDIEventProcessor.java

```diff
@@ -310,6 +296,41 @@ private void createStepRequest(long threadId, int stepType) {
         request.enable();
     }
 
+    /**
+     * Evaluates the given breakpoint condition (expression) using the ballerina debugger expression evaluation engine.
+     *
+     * @param expression breakpoint expression
+     */
```

Comment:
Shall we add comments and description for `@param threadReference` and `@return`

Reply:
Fixed